### PR TITLE
Adds stackoverflow badge into README.md

### PR DESCRIPTION
**Pull request by gurkanindibay**
DESCRIPTION: Adds stackoverflow badge into README.md

### CODE REVIEW

**Review by metdos — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 [![Slack Status](http://slack.citusdata.com/badge.svg)](https://slack.citusdata.com)
 [![Latest Docs](https://img.shields.io/badge/docs-latest-brightgreen.svg)](https://docs.citusdata.com/)
 [![Code Coverage](https://codecov.io/gh/citusdata/citus/branch/master/graph/badge.svg)](https://app.codecov.io/gh/citusdata/citus)
+<a href="https://stackoverflow.com/questions/tagged/citus">
```
[the remark this answers is not in the document]
Comment (gurkanindibay) [reply README.md]: I fixed formats similar to the previous ones

File: README.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
 [![Slack Status](http://slack.citusdata.com/badge.svg)](https://slack.citusdata.com)
 [![Latest Docs](https://img.shields.io/badge/docs-latest-brightgreen.svg)](https://docs.citusdata.com/)
 [![Code Coverage](https://codecov.io/gh/citusdata/citus/branch/master/graph/badge.svg)](https://app.codecov.io/gh/citusdata/citus)
+[![Stack Overflow](https://img.shields.io/badge/Stack%20Overflow-%20-545353?logo=Stack%20Overflow)](https://stackoverflow.com/questions/tagged/citus)
```
Comment (metdos) [inline README.md:6]: Can we find a badge from the Stackoverflow website, we're not pulling any data via img.shields.io, do we need it?

File: README.md [hunk 3]
```diff
@@ -3,6 +3,8 @@
...
+
```
Comment (metdos) [inline README.md:7]: Could you drop this extra line and squash commits?

Comment (gurkanindibay) [reply README.md:7]: Removed the extra line

Comment (gurkanindibay) [reply README.md:7]: @metdos Could you check and approve for me to be able to merge the PR

Comment (metdos) [reply README.md:7]: "squash commits?"

Could you do this first? I want to see only one commit.

Comment (gurkanindibay) [reply README.md:7]: Squashed as you suggested

**Comment by metdos** [comment]
Looks good.

**Review by metdos — APPROVED** [review]

**Pull request merged**